src/main/java/org/jboss/forge/arquillian/container/model/Container.java

```diff
@@ -167,6 +167,12 @@ public String toString() {
         return getDisplayName();
     }
 
+    public boolean isPayaraORGlassFishEmbedded() {
```

Comment:
Do we really have to lock it to certain container names? Isn't it just 'unsupportedContainer`?